packages/mds-agency/tests/request-handlers.spec.ts

```diff
@@ -150,38 +149,13 @@ describe('Agency API request handlers', () => {
       Sinon.replace(cache, 'writeDevices', Sinon.fake.rejects('it-broke'))
       Sinon.replace(stream, 'writeDevice', Sinon.fake.resolves('it-worked'))
       await registerVehicle({ body } as AgencyApiRegisterVehicleRequest, res)
-      assert.equal(statusHandler.calledWith(201), true)
-      assert.equal(sendHandler.called, true)
+      expect(statusHandler.calledWith(201)).toBeTruthy()
+      expect(sendHandler.called).toBeTruthy()
       Sinon.restore()
     })
   })
 
   describe('Get vehicle by id', () => {
```

Comment:
Why cut this test?

Reply:
I talked with @avatarneil and he says this test is worthless, and was causing timeouts, unclosed handles.